engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/ZeroKeyUpdateByManager.java

```diff
@@ -70,7 +69,7 @@ public class ZeroKeyUpdateByManager extends UpdateBy {
 
             // create an updateby bucket instance directly from the source table
             zeroKeyUpdateBy = new UpdateByBucketHelper(bucketDescription, source, windows, resultSources,
-                    timestampColumnName, control, (oe, se) -> deliverUpdateError(oe, se, true));
+                    timestampColumnName, control, (oe, se) -> deliverUpdateError(oe, se, true), new Object[0]);
```

Comment:
Use `io.deephaven.util.type.ArrayTypeUtils#EMPTY_OBJECT_ARRAY` instead, here and L90?